applications/crossbar/priv/couchdb/schemas/devices.json

```diff
@@ -406,6 +406,11 @@
             },
             "type": "object"
         },
+        "rtcp_mux": {
+            "default": true,
```

Comment:
As I write before FreeSwitch default value is `true`. You can check this if you (enable RTCP)[https://wiki.freeswitch.org/wiki/RTCP] at sip profile or as channel var.
Also please look (FS-10400)[https://freeswitch.org/jira/browse/FS-10400] call logs and Mike comment.